src/mock/oppgave-mock.ts

```diff
@@ -1,12 +1,32 @@
 import { Oppgave } from '../models/oppgave';
+import faker from 'faker/locale/nb_NO';
 import navfaker from 'nav-faker';
+import { MOCKED_TRAADID_1, MOCKED_TRAADID_2, MOCKED_TRAADID_3 } from './meldinger/meldinger-mock';
 
 export function getTilfeldigeOppgaver(): Oppgave[] {
     if (navfaker.random.vektetSjanse(0.05)) {
         return [];
     }
 
-    return Array.from({ length: navfaker.random.integer(4, 1) }, () => {
-        return { fødselsnummer: navfaker.personIdentifikator.fødselsnummer(), henvendelseid: '123', oppgaveid: '456' };
+    const fodselsnummer = navfaker.personIdentifikator.fødselsnummer();
+
+    const oppgaveArray = Array.from({ length: navfaker.random.integer(3, 1) }, () => {
```

Comment:
Dette var litt magi for meg :-P

Hva med å bruke den flotte metoden du lagde: `fyllRandomListe`